literatur.bib

```diff
@@ -12445,6 +12517,13 @@ @Misc{olmo-et-al-arxiv2021
   howpublished = "arXiv:2106.07131 [cs.CL]"
 }
 
+@InProceedings{olz-bercher-icaps2019,
+  author =       "Conny Olz and Pascal Bercher",
+  title =        "Eliminating Redundant Actions in Partially Ordered Plans -- A Complexity Analysis",
```

Comment:
```suggestion
  title =        "Eliminating Redundant Actions in Partially Ordered Plans --- A Complexity Analysis",
```

Reply:
That's debatable. I think many modern styles prefer "--" over "---" everywhere.

Reply:
That doesn't really matter, this is the kind of typesetting convention that isn't supposed to be "exported" from the original title to the citing paper, but the conventions of the citing paper apply. (In other words, the goal should be consistency on this within the bib file.) Not that I think we're currently 100% consistent.

Reply:
Good to know, I didn't have that on my radar.